client/src/components/Header.js

```diff
@@ -210,8 +211,13 @@ export default class Header extends Component {
     }
 
     renderLogo() {
+        let imageLink = this.props.loggedIn ? this.props.schoolLogo : require('./logo.png')
         return (
-            <img style={{'paddingTop': '30px'}} className="ui small image centered" src={require("./logo.png")} alt="logo"/>
+            <Grid
+                centered
+            >
+                <Image src={imageLink} size='small'/>                
```

Comment:
Image may not be centered properly within grid - Images have a centered tag that should align them properly